app/assets/javascripts/discourse/controllers/invite.js.es6

```diff
@@ -54,6 +54,24 @@ export default ObjectController.extend(ModalFunctionality, {
   isPrivateTopic: Em.computed.and('invitingToTopic', 'model.category.read_restricted'),
 
   /**
+    Allow Existing Members? (username autocomplete)
+
+    @property allowExistingMembers
+  **/
+  allowExistingMembers: function() {
+    return this.get('invitingToTopic') && !this.get('isPrivateTopic');
+  }.property('invitingToTopic', 'isPrivateTopic'),
+
+  /**
+    Show Groups? (add invited user to private group)
+
+    @property showGroups
+  **/
+  showGroups: function() {
+    return this.get('isAdmin') && !this.get('allowExistingMembers');
```

Comment:
this makes no sense, you are stopping admins from inviting into existing groups, this should only happen AFTER they select a user that is not an email